vita3k/gui/src/pkg_install_dialog.cpp

```diff
@@ -102,12 +105,16 @@ void draw_pkg_install_dialog(GuiState &gui, HostState &host) {
             if (ImGui::Button("Ok", BUTTON_SIZE) && !zRIF.empty())
                 state = "install";
         } else if (state == "install") {
-            if (install_pkg(std::string(pkg_path), host, zRIF))
-                state = "succes";
-            else
-                state = "fail";
-            zRIF.clear();
-        } else if (state == "succes") {
+            std::thread installation(([&host]() {
```

Comment:
why here have double "("  after installation ? other have only one 🤔

Reply:
Lol I must have just typed it twice